pajbot/modules/wolfram.py

```diff
@@ -193,6 +199,7 @@ def load_commands(self, **options):
             level=self.settings["level"],
             description="Ask Wolfram Alpha a question",
             command="query",
+            can_execute_with_whisper=bool(self.settings["response_method"] == "reply"),
```

Comment:
I don't think you need the `bool()` here, last time I checked the result of `==` was always a bool.

Reply:
So I just enclose it in brackets?